insonmnia/dwh/storage.go

```diff
@@ -56,6 +56,9 @@ type storage interface {
 	GetLastKnownBlock(conn queryConn) (uint64, error)
 	InsertLastKnownBlock(conn queryConn, blockNumber int64) error
 	UpdateLastKnownBlock(conn queryConn, blockNumber int64) error
+	StoreID(conn queryConn, id *big.Int, entity string) error
+	RemoveID(conn queryConn, id *big.Int, entity string) error
+	CheckID(conn queryConn, id *big.Int, entity string) (bool, error)
```

Comment:
These three methods names are not really obvious, names explain actions, but not an entity to act with. 
I'd personally prefer to make it bit more self-describing (like all previous methods do).